.github/workflows/tests.yml

```diff
@@ -50,16 +50,21 @@ jobs:
       - name: Setup dependency cache
         id: composercache
         run: echo "::set-output name=dir::$(composer config cache-files-dir)"
-        
+
       - name: Cache dependencies
         uses: actions/cache@v2
         with:
           path: ${{ steps.composercache.outputs.dir }}
           key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
           restore-keys: ${{ runner.os }}-composer-
 
-      - name: Install Composer dependencies
-        run: composer install --no-interaction --no-progress --no-suggest
+      - name: Install Composer dependencies (< 8.0)
+        if: matrix.phpVersion < '8.0'
+        run: composer install --no-interaction --no-progress --no-suggest --no-scripts
+
+      - name: Install Composer dependencies (8.0)
+        if: matrix.phpVersion == '8.0'
+        run: composer install --no-interaction --no-progress --no-suggest --no-scripts --ignore-platform-reqs
```

Comment:
Package [rdohms/phpunit-arraysubset-asserts](https://github.com/rdohms/phpunit-arraysubset-asserts) either supports PHP `^7.3|^8.0` and phpunit `^9.0` only, or PHP `^7.2` and phpunit  `^8.4`. 

So it is impossible to use it with PHP 8 and phpunit 8. However it works fine, so this change fixes install failure.